### PR TITLE
✨ 135 - collaborator removed email

### PR DESCRIPTION
**Pull request by anncatton**
Email content and trigger for collaborators removed from an approved application.
Changes that are just formatting/typo fix have comments.

### CODE REVIEW

File: src/emails/collaborator-added.ts [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
+  uiLinksInfo: UILinksInfo,
+) {
+  const emailMjml = compose(
+    {
+      message: messageBody(app, uiLinksInfo, formatName(collaborator.info), collaborator.addedOn),
+      receiver: {
+        first: reviewerInfo.firstName,
+        last: reviewerInfo.lastName,
+      },
+      includeClosure: false,
```
Comment (anncatton) [inline src/emails/collaborator-added.ts:33]: typo fix and formatting only

File: src/emails/ethics-letter.ts [hunk 2]
```diff
@@ -1,26 +1,37 @@
...
+  uiLinksInfo: UILinksInfo,
+) {
+  const emailMjml = compose(
+    {
+      message: messageBody(app, uiLinksInfo, letterInfo),
+      receiver: {
+        first: reviewerInfo.firstName,
+        last: reviewerInfo.lastName,
+      },
+      includeClosure: false,
```
Comment (anncatton) [inline src/emails/ethics-letter.ts:31]: typo fix and formatting only

File: src/emails/review-revised.ts [hunk 3]
```diff
@@ -3,20 +3,25 @@ import { Application } from '../domain/interface';
...
+  uiLinksInfo: UILinksInfo,
+) {
+  const emailMjml = compose(
+    {
+      message: messageBody(app, uiLinksInfo),
+      receiver: {
+        first: reviewerInfo.firstName,
+        last: reviewerInfo.lastName,
+      },
+      includeClosure: false,
```
Comment (anncatton) [inline src/emails/review-revised.ts:21]: typo fix and formatting only

**Review by caravinci — APPROVED** [review]
✉️

**Pull request merged**